splunk-quarkus/README.md

```diff
@@ -35,7 +35,7 @@ it running in background (detached).
 To run it locally with dev mode execute:
 
 ```
-$ ACG_CONFIG=./devel.json ../mvnw quarkus:dev -Dquarkus.kafka.devservices.enabled=false
+$ ACG_CONFIG=./devel.json ../mvnw quarkus:dev -Dquarkus.kafka.devservices.enabled=false -Dquarkus.log.cloudwatch.enabled=false
```

Comment:
this can be removed when you add this to the application.properties in splunk-quarkus/src/main/resources/application.properties:

```
# By default the quarkus cloudwatch logging extension is enabled
# With this property we disable it so we can just use the Quarkus dev mode as usual.
%dev.quarkus.log.cloudwatch.enabled=false
```

Reply:
Added. Thank you!